python/src/util.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from os import path
+
+
+def cell_input():
+	cell_names = [""]
+	replaced = False
+
+	file_path = input( "Enter path to file:")
+	file_name = input( "Enter file name:")
+	cell_names_str = input( "Enter cell names, separted by comma, no space:" )
+
+	# Use if there can only be uppercase lettered cells
+	cell_names_str.upper()
+
+	if cell_names_str is not None:
+		cell_names = cell_names_str.split( ',' )
+
+	old_path = f"{file_path}/bk_{file_name}"
+	new_path = f"{file_path}/{file_name}"
+
+	if path.exists(new_path):
```

Comment:
I like that you added a command line interface. Can we abstract this as a function too, since that is how it will need to be used in the final product.

Reply:
Do you mean to abstract the file parameter input as a separate function?

Reply:
yeah exactly

Reply:
Yeah, ezpz change, did you want it as another function within `util.py` or elsewhere?

Reply:
@jameswolfeoliver might have some thoughts on how he was structuring things, but my thought would be to keep it in the same file. James also would probably know more about how to build pylint locally to figure out those issues